rustls/src/server/tls12.rs

```diff
@@ -422,10 +439,11 @@ mod client_hello {
     }
 
     fn emit_certificate_req(
+        config: &ServerConfig,
         transcript: &mut HandshakeHash,
         cx: &mut ServerContext<'_>,
     ) -> Result<bool, Error> {
-        let client_auth = cx.config.get_verifier();
+        let client_auth = config.get_verifier();
```

Comment:
Seems strange we got rid of the `get_verifier()` for the other side but not here. I think we should revert the other side to use `get_verifier()` like before.

Reply:
Done.